en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown

```diff
@@ -0,0 +1,103 @@
+
+2.  Back up the application specific indexes for Workflow Metrics and Result
+    Rankings.
+
+3. In @product@-ver@, the X-Pack module is now provided out of the box. If
+   you're not X-Pack security, make sure you disable it (it's disabled by
+   default):
+
+    ```yml
+    xpack.security.enabled: false
+    ```
+
+4.  Blacklist the bundled Liferay Connector to Elasticsearch 6. The [Installing Solr guide]()
+
+5.  Install and configure the Liferay Connector to Elasticsearch 7.
+
+6.  Re-index all search  and spell check indexes.
+
+| **Before Proceeding,** back up your existing data before upgrading
+| Elasticsearch. If something goes wrong during or after the upgrade, roll back
+| to the previous version using the uncorrupted index snapshots. See
+| [here](/docs/7-2/deploy/-/knowledge_base/d/backing-up-elasticsearch) for more
+| information.
+
+| **Known Issue:** See
+| [LPS-103938](https://issues.liferay.com/browse/LPS-103938). The Liferay
+| Connector to Elasticsearch 7 will throw an exception in the log when the LPKG
+| file is deployed. There are no known functional impacts. If unexpected errors
+| occur, re-start the @product@ server.
+
+Learn about configuring Elasticsearch [here](/docs/7-2/deploy/-/knowledge_base/d/configuring-the-liferay-elasticsearch-connector).
+
+## Backing up Application-Specific Indexes
+
+To preserve data stored in application-specific indexes, use a
+[rolling upgrade](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/rolling-upgrades.html)
+for each index you need to preserve across the upgrade.
+
+| **Synonym Sets:** If you follow the workaround for the bug
+| [LPS-100272](https://issues.liferay.com/browse/LPS-100272), your Synonym sets
+| are preserved across the upgrade, as they are stored in the index settings
+| directly, and not in their own index.
+
+## Blacklisting Elasticsearch 6
+
+To blacklist Elasticsearch 6,
+
+1.  Create a configuration file named
+
+    ```sh
+    com.liferay.portal.bundle.blacklist.internal.BundleBlacklistConfiguration.config
+    ```
+
+2.  Give it these contents:
+
+    ```properties
+    blacklistBundleSymbolicNames=["com.liferay.portal.search.elasticsearch6.api","com.liferay.portal.search.elasticsearch6.impl","com.liferay.portal.search.elasticsearch6.spi","Liferay Connector to X-Pack Security [Elastic Stack 6.x]"]
+    ```
+
+## Re-index
+
+Once the Elasticsearch adapter is installed and talking to the Elasticsearch
+cluster, navigate to *Control Panel* &rarr; *Configuration* &rarr; *Search*,
+and click *Execute* for the *Reindex all search indexes* entry.
+
+You must also re-index the spell check indexes.
+
+## Reverting to Elasticsearch 6
+
+Stuff happens. If that stuff involves an unrecoverable failure during the
+upgrade to Elasticsearch 7, roll back to Elasticsearch 6 and regroup.
+
+Since your Elasticsearch 6 and 7 are currently two separate installations, this procedure
```

Comment:
this procedure takes only a few steps: 

(we never, ever say something is "straightforward") https://github.com/liferay/liferay-docs/blob/master/guidelines/03-writers-guidelines.markdown#common-documentation-and-phraseology-issues